### ISSUE TITLE
Daily backups?

### ISSUE THREAD

**Issue by sjf-control** [matthewhadley/homeassistant-dropback#9]
I just wanted to verify that by itself this Add-on will not generate or copy any backups. That the user must periodically run the the script you've provided in the "Usage" section to trigger the copying of whatever backups are currently existing. He must also periodically trigger those backups to be generated. Correct?

**Comment by matthewhadley** [comment]
Correct. The add-on only takes action when the service call is made.

In my own setup, I have an automation that:
- runs at 3am
- performs a full backup
- waits 30 mins for that to complete (this could probably be a lot less time)
- calls the add-on to sync to dropbox

Here is the YAML
```
alias: System Backup To Dropbox
description: ""
trigger:
  - platform: time
    at: "03:00:00"
action:
  - service: hassio.backup_full
    data:
      name: "{{ now().strftime('%Y-%m-%d') }}"
  - delay: "00:30:00"
  - service: hassio.addon_stdin
    data:
      addon: 719b45ef_dropback
      input: sync
mode: single

```

**Comment by sjf-control** [comment]
Thanks for that Matt. I figured it’d be something like that, but now I don’t need to look it up. You might want to add that to your docs. 
in addition, the most difficult part of the installation was step 5. Could not find the submit button. There was a “revert submit” button, but no “submit” button. Turns out it was two buttons side-by-side. At least here i could not see that it was two separate buttons.
Thanks for your help!
